config/newrelic.yml

```diff
@@ -58,6 +58,7 @@ test:
 staging:
   <<: *default_settings
   app_name: <%= ENV.fetch('NEW_RELIC_APP_NAME', 'Chatwoot')  %> (Staging)
+  monitor_mode: <%= ENV.fetch('NEW_RELIC_STAGING_MONITORING_ENABLED', false) %>
```

Comment:
We can have `NEW_RELIC_MONITORING_ENABLED` as the environment variable and also have this option passed in for test environment as well.

Reply:
Makes sense. I will close out this PR and make the change in the original PR. 

https://github.com/chatwoot/chatwoot/pull/10408